src/test/java/org/jenkinsci/plugins/workflow/job/WorkflowRunRestartTest.java

```diff
@@ -270,7 +270,7 @@
 
     private boolean hasTermOrKillLink(WorkflowRun b, String termOrKill) throws Exception {
         return !story.j.createWebClient().getPage(b)
-                .getByXPath("//a[@href = '#' and contains(@onclick, '/" + b.getUrl() + termOrKill + "')]").isEmpty();
+                .getByXPath("//a[@href = '#' and contains(@data-url, '/" + b.getUrl() + termOrKill + "')]").isEmpty();
```

Comment:
Behaviour changed in https://github.com/jenkinsci/jenkins/pull/5515 in 2.320

Not picked up in bom due to https://github.com/jenkinsci/bom/blob/231c8d6403cf235a1f149fd24db857395d510cad/excludes.txt#L16-L17